jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -131,13 +140,18 @@ private void executeTimerJobInstance(TimerJobInstance timerJobInstance) throws E
         }
     }
 
-    private void recoverTimerJobInstance(EjbTimerJob ejbTimerJob, Exception e) {
+    private void recoverTimerJobInstance(Timer timer, EjbTimerJob ejbTimerJob, Exception e) {
```

Comment:
timer is not used in recoverTimeJobInstance

Reply:
removed